x-pack/test/apm_api_integration/tests/services/annotations.spec.ts

```diff
@@ -79,52 +109,49 @@ export default function annotationApiTests({ getService }: FtrProviderContext) {
       });
 
       it('fails with a 400 bad request if data is missing', async () => {
-        const response = await request({
-          url: '/api/apm/services/opbeans-java/annotation',
-          method: 'POST',
-        });
+        try {
+          await createAnnotation();
+        } catch (error: unknown) {
+          const apiError = error as ApmApiError;
 
-        expect(response.status).to.be(400);
+          expect(apiError.res.status).eql(400);
```

Comment:
Like @gbamparop mentioned it would be beneficial to use `expectToReject` here. Jest has a `expect.assertions(2);` to verify the number of assertions made but I'm not sure Mocha has anything equivalent